src/i_winmusic.c

```diff
@@ -76,6 +76,15 @@ static byte channel_volume[MIDI_CHANNELS_PER_TRACK];
 static float volume_factor = 0.0f;
 static boolean update_volume = false;
 
+typedef enum
+{
+    STATE_PLAYING,
+    STATE_STOPPED,
+    STATE_PAUSED
+} player_state_t;
+
+static player_state_t player_state;
```

Comment:
I think this would be better called `music_player_state` to signal that it doesn't actually affect gameplay. Also the enum name.

Should `player_state` have an explicit default value?

Reply:
https://github.com/chocolate-doom/chocolate-doom/pull/1603/commits/9bca206dae5fd38768fc4434175cf92c5d6b095b and https://github.com/chocolate-doom/chocolate-doom/pull/1603/commits/b6ab88a65d9604518213526440dbfb21377f462f